### ISSUE TITLE
[Enhancement] Create a fork of an online editor with Kirby installed

### ISSUE THREAD

**Issue by kodeaben** [kirbydesign/designsystem#2717]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Many issues reported on open source repos includes a link to an online editor where the bug reporter has recreated the bug he/she experiences. This makes it easier for the maintainers of the repo to understand the bug and find a fix. 

An example of an online editor could be [stackBlitz](https://stackblitz.com/).

## Describe the solution you'd like
 - [ ] I'd like a fork of a clean Angular project, in which the Kirby packages are already installed. We will also have to create new forks whenever we release a new version. This will also make is easy to test bugs i different versions of Kirby.
 - [ ] A link from the cookbook to this fork and a page with a list of all versions of kirby.
 - [ ] The `Kirby guild` bot should promote this fork to the users so they are aware of it when they write about bugs

## Have you considered any alternatives?
No
## Are there any additional context?
 This could also be an elegant way to show examples of Kirby components in the cookbook. (Out of scope for this issue)
<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [-] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [-] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
